UPGRADING.asciidoc

```diff
@@ -107,6 +107,17 @@ This feature has been disabled by default. If you have enabled the feature the c
 
 ### Removed configuration settings
 
+#### Index Retention and Rotation Settings
+
+In 2.0 the index rotation and retention settings have been moved from the Graylog server config file to the database and are
+now configurable via the web interface.
+
+The old settings from the `graylog.conf` or `/etc/graylog/server/server.conf` will be migrated to the database.
+
+**WARNING:** When you upgrade from a 1.x version and you modified any rotation/retention settings, please make sure you **KEEP** your old settings in the config file so the migration process will add your old settings to the database! Otherwise the retention process will use the default settings and might remove a lot of indices.
```

Comment:
Maybe use the standard admonition `WARNING:` here?
http://asciidoctor.org/docs/user-manual/#admonition